Dockerfile_init

```diff
@@ -5,7 +5,7 @@ MAINTAINER PhenoMeNal-H2020 Project <phenomenal-h2020-users@googlegroups.com>
 LABEL Description="Galaxy 18.01-pheno-dev for running inside Kubernetes."
 LABEL software="Galaxy Init"
 LABEL software.version="18.01-pheno-dev"
-LABEL version="1.5"
+LABEL version="1.7"
```

Comment:
let's leave the container major.minor version unchanged in this PR.  We'll bump it up as we do the RC